### ISSUE TITLE
/dial_peers? response status 404

### ISSUE THREAD

**Issue by Frederic-Zhou** [tendermint/tendermint#7490]
Hi!

I installed tendermint v0.35.0，and start with --proxy-app=kvstore,but the `/dial_seeds` endpoint of the RPC response status 404!
<img width="388" alt="image" src="https://user-images.githubusercontent.com/3068492/147244661-1da83b7c-0a30-4520-8f9f-01a49249f7d5.png">

**Comment by Frederic-Zhou** [comment]
I've solved this problem.
`--rpc.unsafe=true`

**Comment by Frederic-Zhou** [comment]
I found that `/dial_peers` has been removed in commit[03ad7d6f20d9fd8fe83d31db168f433480552e94](https://github.com/tendermint/tendermint/commit/03ad7d6f20d9fd8fe83d31db168f433480552e94#diff-0ecc577f27d8dd66129dc2f8fb18ae69cd524953fc2217ea46ada31bfd694147 )

Is there any other way to dynamically join peers?

**Comment by creachadair** [comment]
Can you say more about what problem you're attempting to solve using `/dial_peers`?

Note that commit 03ad7d6f is not on the release branch for v0.35, so if you're building from v0.35.0 that change should not affect your build.

**Comment by Frederic-Zhou** [comment]
I build two versions of the planet, 0.35.0 and the latest master branch version
Actually, there's nothing problem with using 0.35.0,I just curious if the main branch version removes /dial_peers because there are better alternatives?

**Comment by creachadair** [comment]
> I build two versions of both 0.35.0 and the latest master branch version Actually, there's nothing problem with using 0.35.0,I just curious if the main branch version removes /dial_peers because there are better alternatives?

As the "unsafe" designation suggests, this endpoint isn't recommended for production use. But I can't recommend a better alternative without knowing more about what problem you're hoping to solve (since "better" depends on the context 🙂).

Generally speaking, manually adjusting network settings at runtime is risky—it would be very easy to DoS your own network, and having the controls active exposes another attack surface.

**Comment by Frederic-Zhou** [comment]
Hello，@creachadair 
Therefore, the recommended safe networking method is to set the address of the peer node in the configuration file before running, right?

**Comment by webmaster128** [comment]
> Can you say more about what problem you're attempting to solve using `/dial_peers`?

I just came here from Google as I followed [this docs](https://docs.tendermint.com/v0.34/tendermint-core/using-tendermint.html) and got the 404.

The use case is this: I want to add a few initial peers manually to my own node that are (a) not seeds and (b) not persistent. I need some starting point for the PeX system. I thought /dial_peers could be the solution. Is there a CLI alternative that does not require me to manually edit the address book?
